packages/desktop-client/src/hooks/useSelected.tsx

```diff
@@ -246,14 +246,14 @@ export function useSelected<T extends Item>(
 }
 
 const SelectedDispatch = createContext<(action: Actions) => void>(null);
-const SelectedItems = createContext<Set<unknown>>(null);
+const SelectedItems = createContext<Set<string>>(null);
 
 export function useSelectedDispatch() {
   return useContext(SelectedDispatch);
 }
 
-export function useSelectedItems() {
-  return useContext(SelectedItems);
+export function useSelectedItems<T = unknown>() {
+  return useContext<Set<T>>(SelectedItems);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏same here
```suggestion
export function useSelectedItems() {
  return useContext(SelectedItems);
```